scripts/clean-db-dump.sql

```diff
@@ -80,6 +80,26 @@ WHERE supplier_id IN
     (SELECT supplier_id
      FROM dangling_suppliers);
 
+-- Overwrite supplier contact details
+UPDATE
+  contact_information
+SET
+  (
+    contact_name,
+    phone_number,
+    email,
+    address1,
+    city,
+    postcode
+  ) = (
+    'Supplier Contact',
```

Comment:
Can we do something like we do in the users table update and include the ID here so we get some differences between all the entries? Something like `contact_name = 'Supplier Contact #' || id, phone_number = '01234 ' || id, etc etc.

Reply:
How about this:
```
digitalmarketplace=# UPDATE
digitalmarketplace-#   contact_information
digitalmarketplace-# SET
digitalmarketplace-#   (
digitalmarketplace(#     contact_name,
digitalmarketplace(#     phone_number,
digitalmarketplace(#     email,
digitalmarketplace(#     address1,
digitalmarketplace(#     city,
digitalmarketplace(#     postcode
digitalmarketplace(#   ) = (
digitalmarketplace(#     'Supplier #' || supplier_id::TEXT || ' Contact',
digitalmarketplace(#     '555' || supplier_id::TEXT,
digitalmarketplace(#     'simulate-delivered@notifications.service.gov.uk',
digitalmarketplace(#     'Supplier #' || supplier_id :: TEXT || ' Contact Address 1',
digitalmarketplace(#     'Supplier #' || supplier_id :: TEXT || ' Contact City',
digitalmarketplace(#     'AA11 1AA'
digitalmarketplace(# );
UPDATE 7833
digitalmarketplace=# select * from contact_information limit 1
;
  id  | supplier_id |      contact_name       | phone_number |                      email                      |             address1              |             city             | postcode 
------+-------------+-------------------------+--------------+-------------------------------------------------+-----------------------------------+------------------------------+----------
 7169 |       92301 | Supplier #92301 Contact | 55592301     | simulate-delivered@notifications.service.gov.uk | Supplier #92301 Contact Address 1 | Supplier #92301 Contact City | AA11 1AA
(1 row)

```